plugins/Actions/ArchivingHelper.php

```diff
@@ -443,12 +443,20 @@ public static function getActionExplodedNames($name, $type, $urlPrefix = null)
 
         $name = str_replace("\n", "", $name);
 
+        if ($type == Action::TYPE_PAGE_TITLE && self::$actionTitleCategoryDelimiter === '') {
+            if (empty($name)) {
+                $name = self::getUnknownActionName($type);
+            }
+            return array(trim($name));
```

Comment:
Should we do the trim() before checking for an `empty($name)`?

Reply:
done, also realised `empty` but not be best if the name of a title is `0` (which might not happen but who knows)